tensorflow/python/distribute/distribute_lib.py

```diff
@@ -633,19 +639,30 @@ class InputOptions(
       strategy.experimental_distribute_dataset(
           dataset,
           tf.distribute.InputOptions(
-              experimental_prefetch_to_device=False)))
+              experimental_replication_mode=experimental_replication_mode.PER_WORKER,
+              experimental_copy_dataset_on_device=False)))
   ```
 
   Attributes:
     experimental_prefetch_to_device: Boolean. Defaults to True. If True, dataset
       elements will be prefetched to accelerator device memory. When False,
       dataset elements are prefetched to host device memory. Must be False when
       using TPUEmbedding API.
+    experimental_replication_mode: Replication mode for the input function. Currently, the
+      InputReplicationMode.PER_REPLICA is only supported with 
+      tf.distribute.MirroredStrategy.experimental_distribute_datasets_from_function.
+      The default value is InputReplicationMode.PER_WORKER.
+    experimental_copy_dataset_on_device: Boolean. Default to False. When True, dataset
+      will be placed on the device, otherwise it will remain on the host.
```

Comment:
document that experimental_copy_dataset_on_device=True can only be used with experimental_replication_mode=PER_REPLICA

Reply:
added